web/src/views/Apply/FormQuestions.tsx

```diff
@@ -0,0 +1,190 @@
+import Image from "next/image";
+import deleteIcon from "~public/icons/close.svg";
+import addIcon from "~public/icons/plus.svg";
+import {Form, Formik, Field, ErrorMessage, FieldArray} from "formik";
+import * as Yup from 'yup';
+
+const styles = {
+  label: 'block text-gray-700 text-sm font-bold pt-2 pb-1',
+  field:
+    'bg-gray-200 text-gray-700 focus:outline-none focus:shadow-outline border border-gray-300 rounded py-2 px-4 block w-full appearance-none',
+  dynamicField:
+    'bg-gray-200 text-gray-700 focus:outline-none focus:shadow-outline border border-gray-300 rounded py-2 px-4 block w-full appearance-none',
+  button:
+    ' bg-gray-700 text-white font-bold py-2 px-4 w-full rounded hover:bg-gray-600',
+  errorMsg: 'text-red-500 text-sm',
+}
+
+export const FormQuestions = () => {
+  const formSchema = Yup.object().shape({
+    firstName: Yup.string().required('Campo obligatorio'),
+    lastName: Yup.string().required('Campo obligatorio'),
+    email: Yup.string().email('Correo inválido').required('Campo obligatorio'),
+    institution:  Yup.string().required('Campo obligatorio'),
+    links: Yup.array()
+      .of(
+        Yup.object().shape({
+          url: Yup.string().required('Campo obligatorio'),
+        })
+      )
+      .required('Tiene que colocar al menos 1 link'),
+    projectUrl:  Yup.string().required('Campo obligatorio'),
+    reason:  Yup.string().required('Campo obligatorio'),
+    topics:  Yup.string().required('Campo obligatorio'),
+  })
+
+  const initialValues = {
+    firstName: "",
+    lastName: "",
+    email:  "",
+    institution:  "",
+    links: [
+      {
+        url: ''
+      }
+    ],
+    projectUrl: '',
+    reason: '',
+    topics: ''
+  };
+
+  const handleSubmit = (values) => {
+    alert(JSON.stringify(values, null, 2))
+  }
+
+  return (
+    <div className="grid grid-cols-1 items-center gap-y-5">
+      <div>
+        <h1 className="text-3xl font-bold">Datos personales</h1>
+      </div>
+      <Formik
+        initialValues={initialValues}
+        validationSchema={formSchema}
+        onSubmit={handleSubmit}
+      >
+        {({ values }) => (
+          <Form>
+            <div className="grid grid-cols-1 md:grid-cols-2 gap-4 gap-y-5">
+              <div>
+                {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+                <label className={styles.label} htmlFor="firstName">Nombres *</label>
+                <Field className={styles.field} id="firstName" name="firstName"/>
+                <ErrorMessage component='a' className={styles.errorMsg} name='firstName' />
+              </div>
```

Comment:
I think this could be its own component. What do you think?

Reply:
sure